### ISSUE TITLE
.btn with .badge with Glyphicon is taller than other buttons in .btn-group

### ISSUE THREAD

**Issue by legrostdg** [twbs/bootstrap#20096]
On Chromium & Firefox, using v3.3.6, I'm trying to have a glyphicon inside of a badge inside of a btn inside of a btn-group. The btn with the badge is taller ~~bigger~~ than the others:
 
http://jsbin.com/yahasevaro/edit?html,output

    <div class="btn-group">
        <div class="btn btn-default">
          <span class="badge">
            <span class="glyphicon glyphicon-star"></span>
            <span>ba</span>
          </span>
          <span>aa</span>
        </div>
        <div class="btn btn-default">
          ba
        </div>
    </div>

Without a glyphicon, there is no problem.

**Comment by twbs-lmvtfy** [comment]
Hi @legrostdg!

You appear to have posted a live example (http://jsbin.com/yahasevaro/edit), which is always a good first step. However, according to [Bootlint](https://github.com/twbs/bootlint), **your example has some Bootstrap usage errors**, which might potentially be causing your issue:
* line 11, column 5: [E047](https://github.com/twbs/bootlint/wiki/E047): `.btn` should only be used on `<a>`, `<button>`, `<input>`, or `<label>` elements.
* line 18, column 5: [E047](https://github.com/twbs/bootlint/wiki/E047): `.btn` should only be used on `<a>`, `<button>`, `<input>`, or `<label>` elements.
* [W002](https://github.com/twbs/bootlint/wiki/W002): `<head>` is missing X-UA-Compatible `<meta>` tag that disables old IE compatibility modes

You'll need to **fix these errors** and post a revised example before we can proceed further.
Thanks!

(*Please note that this is a [fully automated](https://github.com/cvrebert/lmvtfy) comment.*)

**Comment by legrostdg** [comment]
Please see http://jsbin.com/payatiyuta/1/edit?html,output

**Comment by anuragagarwal561994** [comment]
I tested by just removing the glyphicon element and there is still a difference between heights (slightly lesser though). It is because of the height of the badge itself.
I am not sure if it is a bug but if you need the badge inside the btn-group like this you can try making the display block and adjusting some size/spacing css to achieve the purpose.

**Comment by mdo** [comment]
[**Bootstrap 3 is no longer being officially developed or supported.**](https://github.com/twbs/bootstrap/issues/20631)

All work has moved onto our next major release, v4. As such, this issue or pull request is being closed as a "won't fix." For additional help and support, we recommend [utilizing our community resources](https://github.com/twbs/bootstrap/blob/master/README.md#community). Thanks for your understanding, and see you on the other side of v4!

<3,
@mdo and team
